index.html

```diff
@@ -34,135 +34,206 @@ <h1>
-            <li>Ruby</li>
+            <li class="li-2">RSpec</li>
             &nbsp;&nbsp;&nbsp;
           </ul>
-          <button class="btn">See project</button>
-        </div>
-      </div>
-    </section>
-    <section class="s-card">
-      <div id="card">
-        <div class="card1">
-          <h3>Multi-Post Stories <br />Gain + Glory</h3>
-          <ul class="li">
-            &nbsp;&nbsp;&nbsp;
-            <li>HTML</li>
-            &nbsp;&nbsp;&nbsp;
-            <li>CSS</li>
-            &nbsp;&nbsp;&nbsp;
-            <li>Javascript</li>
+          <br />
+          <ul class="li li-3">
+            <li class="li-2">Capybara</li>
             &nbsp;&nbsp;&nbsp;
-            <li>Ruby</li>
+            <li class="li-2">Selenium</li>
             &nbsp;&nbsp;&nbsp;
           </ul>
-          <button class="btn">See project</button>
         </div>
-      </div>
-    </section>
-    <section class="s-card">
-      <div id="card">
-        <div class="card1">
-          <h3>Multi-Post Stories <br />Gain + Glory</h3>
+      </ul>
+      <ul class="skills center">
+        <div class="skill">
+          <img src="imgs/p3.png" alt="skills" class="p1" />
+          <li><h1 class="s">Skills</h1></li>
           <ul class="li">
             &nbsp;&nbsp;&nbsp;
-            <li>HTML</li>
-            &nbsp;&nbsp;&nbsp;
-            <li>CSS</li>
-            &nbsp;&nbsp;&nbsp;
-            <li>Javascript</li>
+            <li class="li-2">Database Management</li>
             &nbsp;&nbsp;&nbsp;
-            <li>Ruby</li>
+            <li class="li-2">Version Control</li>
             &nbsp;&nbsp;&nbsp;
           </ul>
-          <button class="btn">See project</button>
-        </div>
-      </div>
-    </section>
-    <section class="s-card">
-      <div id="card">
-        <div class="card1">
-          <h3>Multi-Post Stories <br />Gain + Glory</h3>
+          <br />
           <ul class="li">
+            <li class="li-2">CLI</li>
             &nbsp;&nbsp;&nbsp;
-            <li>HTML</li>
+            <li class="li-2">Web Development</li>
             &nbsp;&nbsp;&nbsp;
-            <li>CSS</li>
-            &nbsp;&nbsp;&nbsp;
-            <li>Javascript</li>
-            &nbsp;&nbsp;&nbsp;
-            <li>Ruby</li>
+            <li class="li-2">API Design</li>
             &nbsp;&nbsp;&nbsp;
           </ul>
-          <button class="btn">See project</button>
         </div>
-      </div>
-    </section>
-    <section class="content-3 center">
-      <img class="abt-1" src="imgs/about1.png" />
-      <h1 class="clr-blue">About me</h1>
-      <p class="content-3-p">
-        I can help you build a product , feature or website Look through some of
-        my work and experience! If you like what you see and have a project you
-        need coded, don’t hesitate to contact me.
-      </p>
-      <button class="btn btn-2">Get my resume</button>
-    </section>
-    <section></section>
+      </ul>
+    </div>
```

Comment:
- [ ] Please consider changing this `<div class="skills-div">` to `<ul class="skills-div">`  and `<ul class="skills center">`, `<ul class="skills center frameworks">` to respectivly`<li class="skills center">` and `<li class="skills center frameworks">` as mentionned by the previous reviewer. This is actually what the previous reviewer mean by **"So instead of having separate section element for each category replace it with the li tag and place them in a ul tag"** [this comment](https://github.com/BlesslinJerishR/Microfolio/pull/3#discussion_r1148342131). 
Doing that will make your code way more sementic which will improve the user experience 📱 😄